### PR TITLE
added STS to timestamp rule ignored headers

### PR DESCRIPTION
**Pull request by omerlh**
The following request would raise an alert:
```
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Download-Options: noopen
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
X-Powered-By: Express
Content-Type: application/json; charset=utf-8
Content-Length: 978
```
because of the `max-age`.

### CODE REVIEW

File: test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+
+import static org.junit.Assert.assertEquals;
+
+public class TimestampDisclosureScannerUnitTest extends PassiveScannerTest  {
+    private HttpMessage msg;
+
+    // Hashes in lower case for "guest" without quotes
```
Comment (kingthorin) [inline test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java]: These lines can be removed.

File: test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package org.zaproxy.zap.extension.pscanrulesAlpha;
```
Comment (kingthorin) [inline test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java]: Should have the standard license header.

File: test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java (hunk 1 above)
Comment (omerlh) [reply test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java]: Sorry, should pay more attention when I copied the code...

Comment (kingthorin) [reply test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java]: No worries, been there done that myself :zap:

**Comment by omerlh** [comment]
@thc202 I am not follow how I should update it.
I fixed the other issues.

**Comment by kingthorin** [comment]
@omerlh Add a line to this section: https://github.com/zaproxy/zap-extensions/blob/alpha/src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml#L10

In this case the version # in the file doesn't need to be updated as it hasn't been released (https://github.com/zaproxy/zap-extensions/releases).

**Comment by omerlh** [comment]
Ok, added a comment about the bug fix

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScanner.java [hunk 3]
```diff
@@ -3,7 +3,7 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
- * Copyright 2012 The ZAP development team
+ * Copyright 2017 The ZAP development team
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScanner.java]: We don't update the copyright years on existing classes (not that I want to start a copyright year war, just noting ;)

**Comment by thc202** [comment]
Thanks!

**Pull request merged**